rewrite-maven/src/main/java/org/openrewrite/maven/search/FindProperties.java

```diff
@@ -55,8 +55,11 @@ public String getDescription() {
 
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
-        Pattern propertyMatcher = Pattern.compile(propertyPattern.replace(".", "\\.")
+        Pattern propertyMatcher = Pattern.compile(propertyPattern
+                .replace(".", "\\.")
                 .replace("*", ".*"));
```

Comment:
It looks like we should modify the description of this option, because it really isn't a regular expression at all. It is more like a glob. Then if someone asks for regex support I guess we have to add a boolean option to control whether the pattern is a glob or regex.